CDCgov/cdc-open-viz
# Feature/dev 8823

## Testing Steps
Open the Paired Bar Chart.
Navigate to the "Value Axis" panel and click the "Use Responsive Ticks" checkbox to add responsive logic to ticks that are about to overlap.
Add a value to the "Max Tick Rotation" field to set the maximum rotation when it is responsive. If not specified, it will default to 90 degrees.
Add a prefix or suffix to the ticks to make them longer and observe the result.
Change the browser size to verify functionality across different screen types.
<img width="1475" alt="Screenshot 2024-08-06 at 11 23 13" src="https://github.com/user-attachments/assets/79022367-a6be-4cc4-8481-d7b6f57a7011">




## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
